pkg/agent/clusterclaim.go

```diff
@@ -0,0 +1,97 @@
+package agent
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/stolostron/hypershift-addon-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	clusterclientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+
+	clusterv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+)
+
+const (
+	// labelExcludeBackup is true for the local-cluster will be not backed up into velero
+	labelExcludeBackup = "velero.io/exclude-from-backup"
```

Comment:
@zhiweiyin318 Please help to make sure this is correct?

Reply:
I think it is correct.